src/utilities/gql.ts

```diff
@@ -334,6 +334,17 @@ const gql = {
     }
   `,
 
+  CREATE_ACTIVITY_DIRECTIVES: `#graphql
+    mutation CreateActivityDirectives($activityDirectivesInsertInput: [activity_directive_insert_input!]!) {
+      insert_activity_directive(objects: $activityDirectivesInsertInput) {
```

Comment:
Can you add an `INSERT_ACTIVITY_DIRECTIVES` to the `Queries` enum and use the enum here instead? The `Queries` enum is a way to tie the query to a permission. Maybe these call for an explicit `canBulkCreate` and `canBulkUpdate` feature permission for `activityDirective` in the `permissions.ts` utility file.